lib/board.rb

```diff
@@ -0,0 +1,53 @@
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Metrics/AbcSize
```

Comment:
You should not overuse RuboCop disable cops comments, only complexity ones are allowed to be disabled.